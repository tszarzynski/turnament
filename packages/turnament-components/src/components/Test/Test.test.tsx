import React from 'react';
import { render } from '@testing-library/react'
import Test from './Test';

describe('Test', () => {
   it('renders without error', () => {
      const { asFragment } = render(
         <Test/>
       )
      expect(asFragment()).toMatchSnapshot()
   });
});