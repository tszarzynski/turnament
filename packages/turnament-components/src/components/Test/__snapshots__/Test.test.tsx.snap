// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test renders without error 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    Test
  </div>
</DocumentFragment>
`;
